src/cloud_providers/aws/src/s3.rs

```diff
@@ -196,7 +193,7 @@ impl S3Client {
 
 #[cfg(test)]
 /// Single tests used to pass, but all tests in conjuction didn't due to side effects in the s3 bucket during testing
-/// Due to concurrent test execution in rust, the file cloud_providers/aws/s3.rs caused race conditions and instability in both exporters/s3.rs and cloud_providers/aws/s3.rs due to side effects on the mounted S3 bucket.
+/// Due to concurrent test execution in rust, the file cloud_providers/types/s3.rs caused race conditions and instability in both exporters/s3.rs and cloud_providers/types/s3.rs due to side effects on the mounted S3 bucket.
 /// To fix this, I added cleanup steps before and after each test to maintain a clean state and used the #[serial] attribute to enforce sequential execution, preventing concurrent access.
 pub mod tests {
```

Comment:
@davincios @davincios @blaginin do we still need these tests? We don't export anymore parquet files